include/swift/AST/Builtins.def

```diff
@@ -429,6 +429,12 @@ BUILTIN_SIL_OPERATION(IsUnique, "isUnique", Special)
 /// BridgeObject to be treated as a native object by the runtime.
 BUILTIN_SIL_OPERATION(IsUnique_native, "isUnique_native", Special)
 
+BUILTIN_SIL_OPERATION(BeginCOWmutation, "beginCOWmutation", Special)
```

Comment:
I think this is the primary source of documentation for Builtin's, so we should have really clear comments here.